libs/api/src/lib/graphql/action/action.private-queries.ts

```diff
@@ -138,5 +138,5 @@ export const getActions = async (
     await Promise.all([articles, pages, comments, authors, subscriptions, polls, users, events])
   ).flat()
 
-  return actions.sort((v1: Action, v2: Action) => v2.date.getDate() - v1.date.getDate())
+  return actions.sort((v1: Action, v2: Action) => v2.date.getTime() - v1.date.getTime())
```

Comment:
@Itrulia the sort won't work with Date type so I needed it as a number.

Reply:
Yeah this will make it a number :D

![image](https://user-images.githubusercontent.com/1131901/226836460-52a3edad-a25a-42db-ad2b-af61d236e7d2.png)

Reply:
Actually you're right it should work, but ts shows error.